### PR TITLE
feat: add `OnyxEmpty` component

### PR DESCRIPTION
**Pull request by larsrickert**
Relates to #744

Add empty component.

## Checklist

- [x] The added / edited code has been documented with [JSDoc](https://jsdoc.app/about-getting-started)
- [x] If a new component is added, at least one [Playwright screenshot test](https://github.com/SchwarzIT/onyx/actions/workflows/playwright-screenshots.yml) is added
- [x] A changeset is added with `npx changeset add` if your changes should be released as npm package (because they affect the library usage)

### CODE REVIEW

File: packages/sit-onyx/src/components/OnyxEmpty/OnyxEmpty.stories.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { OnyxLink } from "@/index";
+import { createTruncationDecorator } from "@/utils/storybook";
+import emojiSad from "@sit-onyx/icons/emoji-sad.svg?raw";
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import { h } from "vue";
+import OnyxIcon from "../OnyxIcon/OnyxIcon.vue";
+import OnyxEmpty from "./OnyxEmpty.vue";
+
```
Comment (BoppLi) [inline packages/sit-onyx/src/components/OnyxEmpty/OnyxEmpty.stories.ts:9]: should there be an example with a different icon size? in Figma it is a property, I understand that that is not how it works with the slot but a Dev that inspects the component in the project might search for how it is done.
image

on the other hand teh dev might see in the example that OnyxIcon is used and look into the docs of OnyxIcon ^^

it's fine either way :)

image
when I saw this I first thought the size was about density

File: packages/sit-onyx/src/components/OnyxEmpty/OnyxEmpty.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<script lang="ts" setup>
+import circleX from "@sit-onyx/icons/circle-x.svg?raw";
+import OnyxIcon from "../OnyxIcon/OnyxIcon.vue";
+
```
[the remark this answers is not in the document]
Comment (larsrickert) [reply packages/sit-onyx/src/components/OnyxEmpty/OnyxEmpty.vue:4]: #833

File: packages/sit-onyx/src/components/OnyxEmpty/OnyxEmpty.vue [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+</script>
+
+<template>
+  <div class="onyx-empty">
+    <slot name="icon">
+      <OnyxIcon :icon="circleX" size="48px" />
+    </slot>
+
+    <div class="onyx-empty__label onyx-text onyx-truncation-multiline">
+      <slot></slot>
```
Comment (BoppLi) [inline packages/sit-onyx/src/components/OnyxEmpty/OnyxEmpty.vue:24]: question: was a default text desired from UX?
if not, should / do we want to add a default empty text to our translations that is also used inside the storybook example? could be quite handy to spare the projects from translating that all the time ^^

**Review by BoppLi — CHANGES_REQUESTED** [review]

File: packages/sit-onyx/src/components/OnyxEmpty/OnyxEmpty.ct.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import { mockPlaywrightIcon } from "@/utils/playwright";
+import { expect, test } from "../../playwright-axe";
+import OnyxIcon from "../OnyxIcon/OnyxIcon.vue";
+import OnyxEmpty from "./OnyxEmpty.vue";
+
+test("should render", async ({ mount }) => {
```
Comment (BoppLi) [inline packages/sit-onyx/src/components/OnyxEmpty/OnyxEmpty.ct.tsx:6]: isn't the a11y part missing?

```
  // ACT
  const accessibilityScanResults = await makeAxeBuilder().analyze();

  // ASSERT
  expect(accessibilityScanResults.violations).toEqual([]);
```

**Review by BoppLi — APPROVED** [review]

**Review by BoppLi — DISMISSED** [review]

File: packages/sit-onyx/src/components/OnyxEmpty/OnyxEmpty.vue (hunk 3 above)
Comment (BoppLi) [reply packages/sit-onyx/src/components/OnyxEmpty/OnyxEmpty.vue:24]: we should ask jannick if we want to align it with figma, then the figma component should also have a default text :)
Inside components like the table would also be totally fine ^^

**Review by BoppLi — APPROVED** [review]

**Pull request merged**